### PR TITLE
SC quickstart and troubleshooting

### PR DESCRIPTION
**Pull request by kimsauce**
### Description

Edits based on Nancy's assessment of the SC Quickstart doc (see https://saucedev.atlassian.net/browse/TCD-2116). 

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist

- [x] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/main/docs/contributing.md) document.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docs/secure-connections/sauce-connect/quickstart.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ Get up and running with a Sauce Connect Proxy tunnel in just a few minutes by fo
 
 ## Starting a Tunnel
 
-1. If you haven't yet, download the Sauce Connect Proxy client from [Sauce Connect Proxy Installation](/secure-connections/sauce-connect/installation).
+1. If you haven't yet, download the Sauce Connect Proxy client from [Sauce Connect Proxy Installation](/secure-connections/sauce-connect/installation). If you already have it, confirm you're using the latest version (otherwise, you may run into technical issues).
```
Comment (sweeneyskirt-sl) [inline docs/secure-connections/sauce-connect/quickstart.md:15]: Perfect.

File: docs/secure-connections/sauce-connect/troubleshooting.md [hunk 2]
```diff
@@ -7,11 +7,12 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
-:::note --doctor flag deprecated
+Below are some tips to help troubleshoot Sauce Connect Proxy.
 
-Effective with Sauce Connect Proxy version 4.7.0, we've [deprecated the `--doctor` flag](https://changelog.saucelabs.com/en/sauce-connect-proxy-version-30JTvzO0F). Diagnostics are now automatically triggered.
-:::
+:::caution Important
+Make sure you're using the latest version of Sauce Connect Proxy, which you can download [here](/secure-connections/sauce-connect/installation/). Using older versions may cause technical issues. If you run into an error trying to launch a tunnel, this should be your first step.
```
Comment (sweeneyskirt-sl) [inline docs/secure-connections/sauce-connect/troubleshooting.md:13]: If you already have SC installed, will "downloading" the latest version automatically upgrade it, or is there another process for upgrading? I ended up deleting my previous install and reinstalling from scratch, so let's be clear here about what we are advising. Thanks!

**Review by sweeneyskirt-sl — CHANGES_REQUESTED** [review]
Just a question around how to upgrade.

File: docs/secure-connections/sauce-connect/troubleshooting.md (hunk 2 above)
Comment (kimsauce) [reply docs/secure-connections/sauce-connect/troubleshooting.md:13]: Yes you have to delete the previous install, then install the new version

**Comment by kimsauce** [comment]
> Just a question around how to upgrade.

@sweeneyskirt-sl Thank you for your feedback! Made some more changes -- can you please review when you have a moment?

**Review by sweeneyskirt-sl — APPROVED** [review]

**Pull request merged**